pylintrc

```diff
@@ -0,0 +1,29 @@
+[MESSAGES CONTROL]
+disable = 
+	locally-disabled,
+	locally-enabled,
+	no-init,
+	no-self-use,
+	too-few-public-methods,
+	too-many-ancestors,
+	too-many-arguments,
+	too-many-instance-attributes,
+	too-many-public-methods,
+
+[REPORTS]
+output-format = text
+
+[BASIC]
+good-names = f,i,j,k,db,ex,Run,_,__,js
+no-docstring-rgx = __.*__$|test_.+|setUp$|setUpClass$|tearDown$|tearDownClass$|Meta$
```

Comment:
This makes docstrings optional for test-related methods, which avoids introducing meaningless dosctrings,
just to "check the box" and keep `pylint` from complaining...

This just wastes space repeating itself:

```python
def test_page_loads_properly(self):
    """
    Test that the page loads properly
    """
    pass
```

whereas this is no less informative,
with less verbosity..

```python
def test_page_loads_properly(self):
    pass
```

Granted, many times, the method name alone isn't enough context,
so docstrings can be added when they improve understanding,
but aren't forced to be done dogmatically :)